.github/workflows/release.yml

```diff
@@ -3,6 +3,8 @@
 ---
 name: Release
 
+concurrency: release
```

Comment:
Is this change necessary? Did something break?

Reply:
Nothing broke...but it could now that there is both a `docs` and `release` workflow that each can update the `cli.phylum.io` website. Adding `concurrency: release` to each workflow ensures that only one such workflow can run at a time. See the [GitHub Actions documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency) for more detail.

Reply:
That just seems like a workflow issue tbh. Why wouldn't we just run the `docs` workflow automatically on release, rather than having two places where the same workflow can be run from?

Reply:
> Why wouldn't we just run the `docs` workflow automatically on release, rather than having two places where the same workflow can be run from?

Keeping the `Update-Documentation` job in the `release` workflow ensures that it can be made dependent on the `Release` job. Otherwise, the documentation would be updated _before_ it is known that the release is successful...which could be messy.